urbanopt/geojson-modelica-translator
# feat: add spawn to ETS couplings

#### Any background context you want to provide?
Currently GMT only supports connecting spawn loads to stubbed out heating/cooling sources

#### What does this PR accomplish?
Adds couplings to spawn load and heating cooling ETSes

#### How should this be manually tested?
Run the following tests, open the generated packages in dymola and verify everything looks correct
```
poetry run pytest ./tests/model_connectors/test_spawn_heating.py
poetry run pytest ./tests/model_connectors/test_spawn_cooling.py
poetry run pytest ./tests/model_connectors/test_spawn_district_heating_and_cooling_systems.py
```

#### What are the relevant tickets?
#### Screenshots (if appropriate)
<img width="639" alt="Screen Shot 2021-04-14 at 1 10 50 PM" src="https://user-images.githubusercontent.com/18518728/114766059-2845a080-9d23-11eb-84a4-1a1647dd8011.png">
<img width="642" alt="Screen Shot 2021-04-14 at 1 10 06 PM" src="https://user-images.githubusercontent.com/18518728/114766061-2845a080-9d23-11eb-82c5-c8c730b1948d.png">


<img width="498" alt="Screen Shot 2021-04-14 at 1 11 32 PM" src="https://user-images.githubusercontent.com/18518728/114766053-27147380-9d23-11eb-8ad3-28f76c952470.png">
